src/commands/utils/shell.ts

```diff
@@ -0,0 +1,133 @@
+import { Errorable, getErrorMessage } from "./errorable";
```

Comment:
Using the [shelljs](https://www.npmjs.com/package/shelljs)-based implementation from the K8s extension introduces a chain of dependencies with security vulnerabilities, as well as incompatible type definitions. That might make more sense in the K8s extension, because it uses a lot of _other_ shell functions (`ls`, `cat`, etc.).

Here though, we only care about `exec`, which already has a pretty [simple implementation](https://github.com/shelljs/shelljs/blob/master/src/exec.js). In our case I don't think it's worth bringing in the extra baggage for other shell functions.